src/styles/primitives/colors.css

```diff
@@ -67,4 +67,18 @@
   --ods-color-critical-700: #b31b35;
   --ods-color-critical-800: #901132;
   --ods-color-critical-900: #770a30;
+
+  /* Gradients */
+  --ods-gradient-tl-primary-500-to-800: linear-gradient(
+    135deg,
+    var(--ods-color-primary-500),
+    var(--ods-color-primary-800)
+  );
+
+  --ods-gradient-neutral-900: 31 41 51;
+  --ods-gradient-tl-neutral-900: linear-gradient(
+    135deg,
+    rgb(var(--ods-gradient-neutral-900) / 0.24),
+    rgb(var(--ods-gradient-neutral-900) / 0.96)
+  );
```

Comment:
@RuggeroCino, I ran into an interesting little issue where you can either define color stops for the gradients or go all the way and define the whole gradient as a background image. I went for the latter option because it made more sense in terms of on the fly styling, but it does have its drawbacks. Here, for example, I was unable to find a way to add the transparency using the custom ODS color variable as long as that variable was defined as a hex code (no string concatenation in normal css 🤷🏾‍♂️). I'm not at all happy with this solution; what are your thoughts? Right now it's either we define the colors as RGB values or hardcode the gradient stops. Thoughts?

Reply:
I see, CSS custom properties are hard to use with opacity :/ 
I would discuss this with Selim and see if we really need to use opacity for this gradient, if not we could use a lighter neutral color and have something similar to the primary 500 to 800. Otherwise let's discuss about the alternatives